utils/read_file_test.go

```diff
@@ -0,0 +1,106 @@
+package utils
+
+import (
+	"reflect"
+	"testing"
+)
+
+// Test_ReadFile tests the ReadFile function.
+func Test_ReadFile(t *testing.T) {
```

Comment:
Since we don't need this function, these tests can also be removed. But I want to note something. Here you are actually reading a file from the file system; that should be a red flag. When you are writing unit tests, you are trying to create a controlled environment to execute the tested functionality and check the results. Here, because you are reading an actual file, you are not in control of the environment. 
So, how would you test something like this? You simply don't. This kind of behavior where you have to use outside APIs (in this case, the `os.ReadFile` function) are integration code and should only be in the integration layer, this layer we do not test. In this particular case, our integration layer is our `main.go`. That is why, for example, the parsers receive the `[]byte` with the content of the file being parsed, instead of receiving a file path and reading it themselves. Doing that allows us to exclude the parsers from the integration layer and thus we can test them individually, without making outside API calls.